Gemfile

```diff
@@ -183,7 +183,7 @@ gem 'cancancan'
 # Some helpers for structuring CSS/JavaScript
 # Official version https://github.com/winston/rails_utils/pull/30 is no longer maintained.
 # We also want stricter sanitization.
-gem 'rails_utils', git: 'https://github.com/Coursemology/rails_utils.git'
+# gem 'rails_utils', git: 'https://github.com/Coursemology/rails_utils.git'
```

Comment:
if this gem is no longer required, better to just delete this line (and also the comments above that are explaining why this gem is needed) instead of just commenting it out, for the hygiene purpose of this file